Resources/Prototypes/Entities/Objects/Tools/tools.yml

```diff
@@ -337,6 +337,56 @@
     inHandsOnly: true
     key: enum.RcdUiKey.Key
 
+- type: entity
+  id: RPD
+  parent: [ BaseItem, BaseEngineeringContraband ]
```

Comment:
use rcd as a parent, then you can decrease the number of lines

Reply:
will do